app/coffee/HistoryRedisManager.coffee

```diff
@@ -8,13 +8,9 @@ module.exports = HistoryRedisManager =
 	pushUncompressedHistoryOps: (project_id, doc_id, ops = [], callback = (error, length) ->) ->
 		if ops.length == 0
 			return callback(new Error("cannot push no ops")) # This should never be called with no ops, but protect against a redis error if we sent an empty array to rpush
-		opVersions = ops.map (op) -> op?.v
-		logger.log project_id: project_id, doc_id: doc_id, op_versions: opVersions, "pushing uncompressed history ops"
-		jsonOps = ops.map (op) -> JSON.stringify op
-		async.parallel [
-			(cb) -> rclient.rpush Keys.uncompressedHistoryOps({doc_id}), jsonOps..., cb
-			(cb) -> rclient.sadd Keys.docsWithHistoryOps({project_id}), doc_id, cb
-		], (error, results) ->
+		logger.log project_id: project_id, doc_id: doc_id, "marking doc in project for history ops"
+		rclient.sadd Keys.docsWithHistoryOps({project_id}), doc_id, (error) ->
 			return callback(error) if error?
-			[length, _] = results
-			callback(error, length)
\ No newline at end of file
+			rclient.llen Keys.uncompressedHistoryOps({doc_id}), (error, length) ->
```

Comment:
We can get this length from the call to `updateDocument` (with appropriate plumbing), so there shouldn't be a need for this extra redis call